drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/editor/search/GuidedDecisionTableSearchableElement.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.guided.dtable.client.editor.search;
+
+import java.util.Date;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import com.ait.lienzo.client.core.shape.IDrawable;
+import com.ait.lienzo.client.core.shape.IPrimitive;
+import com.google.gwt.i18n.client.DateTimeFormat;
+import org.drools.workbench.models.guided.dtable.shared.model.DTCellValue52;
+import org.drools.workbench.screens.guided.dtable.client.widget.table.GuidedDecisionTableModellerView;
+import org.kie.workbench.common.services.shared.preferences.ApplicationPreferences;
+import org.kie.workbench.common.widgets.client.search.common.Searchable;
+import org.uberfire.ext.wires.core.grids.client.util.GridHighlightHelper;
+import org.uberfire.ext.wires.core.grids.client.widget.grid.GridWidget;
+import org.uberfire.ext.wires.core.grids.client.widget.layer.impl.GridLienzoPanel;
+import org.uberfire.mvp.Command;
+
+public class GuidedDecisionTableSearchableElement implements Searchable {
+
+    private static final String NONE = "";
+
+    private final DateTimeFormat formatter = getFormat();
+
+    private DTCellValue52 cellValue52;
+
+    private Integer row;
+
+    private Integer column;
+
+    private GuidedDecisionTableModellerView.Presenter modeller;
+
+    public void setCellValue52(final DTCellValue52 cellValue52) {
+        this.cellValue52 = cellValue52;
+    }
+
+    @Override
+    public boolean matches(final String text) {
+        return getValue().toUpperCase().contains(text.toUpperCase());
+    }
+
+    public String getValue() {
+        return convertDTCellValueToString(cellValue52);
+    }
+
+    @Override
+    public Command onFound() {
+        return () -> highlightHelper()
+                .withMinX(getMinX())
+                .withMinY(getMinY())
+                .withPaddingX(200)
```

Comment:
Can you please explain these arbitrary padding values?

Reply:
Fixed. Thanks!